bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CTabFolder.java

```diff
@@ -3835,94 +3835,95 @@ boolean updateItems() {
 }
 
 boolean updateItems (int showIndex) {
-	GC gc = new GC(this);
```

Comment:
I'm not sure if we should apply the new method in places where it causes so large format changes and consequently break the git-blame.
IIRC it is a relatively important source of information in SWT.
@iloveeclipse can you tell, if git-blame should be preserved.

For only a few lines it is fine, I guess.
I think the most interesting applications are outside of SWT where it is already used often in try-finally blocks.